spec/presenters/default_deadline_date_time_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe DefaultDeadlineDateTime do
+  describe '#initialize' do
+    it 'should be N business days later' do
+      t = DcTimePresenter.parse('2016-08-29 15:00')
+      expect(DcTimePresenter.convert(t)).to be_within(1).of(t)
```

Comment:
oh interesting -- why not remove seconds altogether? (have seconds count always be zero coming from `DefaultDeadlineDateDate`)